src/sass/formapp.scss

```diff
@@ -1,8 +1,15 @@
 @import url('https://stackpath.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css');
+@import url('https://bootswatch.com/4/cosmo/bootstrap.min.css');
 body {
   font-family: 'ProximaSoft', 'Droid Sans', 'helvetica neue', 'arial', sans-serif !important;
   padding: 24px;
 }
+.btn.disabled, .btn:disabled {
```

Comment:
how about `.formio-read-only .btn.disabled, .formio-read-only .btn:disabled`. hiding all disabled buttons everywhere seems like it's going to potentially break all sorts of things.

Reply:
.formio-read-only is not in there.

Reply:
well we need to find or do something.  we can't just hide disabled buttons

Reply:
ok so that class exists only in form.io 4.10+ and we're on 4.9.  They're up to 4.12.. probably worth updating.

Reply:
yep